Question 4: On the 50th anniversary of The Brady Bunch, four different magazines asked their readers whether they were fans of the show. The answers were: TV Guide 20% fans Entertainment Weekly 23% fans Pop Culture Today 17% fans SciPhi Phanatic 5% fans

Based on these numbers, what percentage of adult Americans would you estimate were Brady Bunch fans on the 50th anniversary of the show?

Mean Calculation:

In [5]:
#Using Python: mean = sum(df['age']) / len(df['age'])

names = ['TV Guide','Entertainment Weekly','Pop Culture Today', 'SciPhi Phanatic']
fans = [20, 23, 17, 5]
         
mean = sum(fans) / len(fans)
mean


16.25

Question 5, Response

Appproximately 16.25% of adult Americans were Brady Bunch fans on the 50th anniversary of the show.